app/Console/Commands/PollLarajobs.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+
+use App\Events\ClickedJob;
+use App\Models\Larajob;
+use Carbon\Carbon;
+use Native\Laravel\Client\Client;
+use Native\Laravel\Notification;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\Storage;
+
+class PollLarajobs extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'app:poll-larajobs';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Poll Larajobs RSS';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        Log::debug('Polling Larajobs');
+
+        if(!Storage::exists('first_run'))
+            Storage::put('first_run', now()->timestamp);
+
+        $rss_feed = file_get_contents(
+            config('notifier.rss_feed'),
+            false,
+            stream_context_create([
+                'ssl' => [
+                    "verify_peer" => false,
+                    "verify_peer_name" => false,
+                ]
+            ])
```

Comment:
Nice! I was running into a similar issue earlier, good to know about both of those options!